File: src/0674.longest-continuous-increasing-subsequence.674/solution.ts
function findLengthOfLCIS (nums: number[]): number {
    let result = 0;
    let index = 0;
    while (index < nums.length) {
        let count = 1;
        index++;
        while (index < nums.length && nums[index] > nums[index - 1]) {
            count++;
            index++;
        }
        result = Math.max(result, count);
    }
    return result;
}

File: src/0674.longest-continuous-increasing-subsequence.674/solution.java
class Solution {
    public int findLengthOfLCIS(int[] nums) {
        int result = 0;
        int index = 0;
        while (index< nums.length){
            int count = 1;
            index++;
            while (index< nums.length && nums[index-1]<nums[index]){
                index++;
                count++;
            }
            result = Math.max(result,count);
        }
        return result;
    }
}